probe/appclient/app_client_internal_test.go

```diff
@@ -29,7 +29,7 @@ func dummyServer(t *testing.T, expectedToken, expectedID string, expectedVersion
 		}
 
 		if have := r.Header.Get(xfer.ScopeProbeVersionHeader); expectedVersion != have {
-			t.Errorf("want %q, have %q", expectedID, have)
+			t.Errorf("want %q, have %q", expectedVersion, have)
```

Comment:
Good catch, I wonder why the tests were passing before :thinking:

Reply:
Could you please also add a similar check here for the `user-agent` field while you're at it?

Reply:
Oh yea. Sorry forgot to mention this change in the pr

Reply:
The tests are passing because this is just the bit that prints out what it found - the check is on the line before.